packages/xs-vat-worker/package.json

```diff
@@ -49,8 +52,10 @@
     "@agoric/marshal": "^0.3.0",
     "@agoric/promise-kit": "^0.2.0",
     "@agoric/swingset-vat": "^0.11.0",
+    "@agoric/xsnap": "^0.0.0",
```

Comment:
Let's line these up to start on 0.1.0 so ^0.1.0 gives us both breaking and non-breaking subsequent numbers.